packages/cysync-core/src/pages/OnBoarding/SetPassword/Dialogs/PasswordForm.tsx

```diff
@@ -0,0 +1,116 @@
+import {
+  PasswordInput,
+  QuestionMarkButton,
+  DialogBoxFooter,
+  Button,
+  Typography,
+  DialogBox,
+  DialogBoxBody,
+  LangDisplay,
+  Flex,
+  Container,
+} from '@cypherock/cysync-ui';
+import React, { useState } from 'react';
+import { routes } from '~/constants';
+import { useNavigateTo } from '~/hooks';
+
+import { useAppSelector, selectLanguage } from '~/store';
+import { validatePassword } from '~/utils';
+import { changePassword } from '~/utils/password';
```

Comment:
Use `setPassword` function from `useLockscreen` hook. Do not use the password utility directly. Maybe add a comment in `utils/password` for future reference.